src/UXCapture.js

```diff
@@ -8,35 +9,49 @@ let _onMeasure;
 let _view;
 
 const UXCapture = {
+	_clearMarksAndMeasures: () => {
+		window.performance.clearMarks();
+		window.performance.clearMeasures();
+	},
+
 	/**
 	 * Sets `onMark` and `onMeasure` callbacks on UXCapture singleton
+	 * Also resets the view (supposed to be called once per page anyway)
 	 *
 	 * @param {object} config
 	 */
-	create: (config) => {
+	create: config => {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
+		_view = null;
```

Comment:
pretty minor, but I think `_view = undefined` might be better here to keep the data type as `View | undefined` rather than `View | undefined | null`